platform-coop-toolkit/coop-map-directory-index
# feat: add 'made by a cooperative' field to tools model

## Description

This PR adds a new choices field, `coop_made`, to the Tools model, which indicates whether or not the tool was made by a cooperative.

## Steps to test

1. Run migrations.
2. Edit a tool from the admin.
3. Visit the tools API endpoint.

**Expected behavior:** In the tool editing interface, a checkbox for "Made by a cooperative" appears. In the API, the `coop_made` attribute appears as `false` if not set, or one of the following if set:

- `unknown`
- `yes`
- `no`

## Additional information

Not applicable.

## Related issues

Not applicable.
